Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/PropertyEditorToolsSystemInterface.h

```diff
@@ -60,6 +60,18 @@ namespace AzToolsFramework
         //! Unregisters a previously registered property handler.
         virtual void UnregisterHandler(PropertyHandlerId handlerId) = 0;
 
+        template<class, class = void>
+        struct HandlerTraits
+        {
+            static constexpr bool IsDefaultHandler = false;
+        };
+
+        template<class T>
+        struct HandlerTraits<T, AZStd::void_t<typename T::IsDefaultHandler>>
+        {
+            static constexpr bool IsDefaultHandler = T::IsDefaultHandler();
+        };
```

Comment:
FYI you can use variable templates for this instead of a class template.

```suggestion
        template<class, class = void>
        constexpr bool IsDefaultHandler = false;
        template<class T>
        constexpr bool HandlerTraits<T, AZStd::void_t<typename T::IsDefaultHandler>> = T::IsDefaultHandler();
```

Reply:
Good call, done! Gosh is this so much nicer than the way SFINAE used to go back in the day...